utils/config/daos_agent.yml

```diff
@@ -0,0 +1,49 @@
+# DAOS client configuration file.
```

Comment:
I thought we were aiming for a single client yml config file, rather than having a separate one for daos agent.  Was there a discussion about this?

Reply:
My understanding from Johann was that we would support a single config format but the dmg tool and the agent would have their own physical config files.